client/css/style.css

```diff
@@ -1557,62 +1549,22 @@ button,
 	font-style: italic;
 }
 
-/**
- * TrackpadScrollEmulator
- * Version: 1.0.6
- * Author: Jonathan Nicol @f6design
- * https://github.com/jnicol/trackpad-scroll-emulator
- */
-.tse-scrollable {
-	position: relative;
-	overflow: hidden;
-}
-
-.tse-scrollable .tse-scroll-content {
-	overflow: hidden;
-	overflow-y: scroll;
+.scrollable {
+	overflow: auto;
+	visibility: hidden;
```

Comment:
This has an unfortunate side effect, so I suggest removing fiddling with scrollbar visibility and letting the OS decide: scrollbars are set up to always show up on my OS, and this change effectively ignores this setup. Essentially, this is an accessibility issue (accessibility is not only about screen readers ;-) ).